docs/DevGuide/DevGuide.md

```diff
@@ -57,3 +57,8 @@ Assumes a thorough familiarity and fluency in SpECTRE's usage of TMP.
 ### CoordinateMap Guide
 Methods for creating custom coordinate maps are discussed here.
 - \ref redistributing_gridpoints "Methods for redistributing gridpoints"
+
+### Continuous Integration
+Explanations on our automated tests and deployments can be found here.
+
+- \ref dev_guide_automatic_versioning
```

Comment:
should there also be a reference to travis here? I know there's one above, but I could see someone looking in this section for it as well

Reply:
Yes, I'll move it down and update it to refer to GitHub Actions in a separate PR

Reply:
Sounds good, thanks!